src/rpc/methods/f3.rs

```diff
@@ -623,6 +624,39 @@ impl RpcMethod<2> for SignMessage {
     }
 }
 
+pub enum F3ExportLatestSnapshot {}
+
+impl F3ExportLatestSnapshot {
+    pub async fn run(path: String) -> anyhow::Result<Cid> {
+        let client = get_rpc_http_client()?;
+        let mut params = ArrayParams::new();
+        params.insert(path)?;
+        let LotusJson(cid): LotusJson<Cid> = client
+            .request("Filecoin.F3ExportLatestSnapshot", params)
```

Comment:
why is there `"Filecoin.F3ExportLatestSnapshot"` and `"F3.ExportLatestSnapshot"`?